src/context/AppProvider/AppContext.tsx

```diff
@@ -264,12 +264,13 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
 
   useEffect(() => {
     if (portfolioLoadingStatus === 'loading') return
+    if (!isConnected) return
```

Comment:
Don't fetch vp without a wallet, which would result in perma-loading states in e.g LP